data/Audiences.ts

```diff
@@ -3,23 +3,32 @@ interface Audience {
   description: string;
   targeting?: string;
   filter_expression?: string;
+  /**
+   * A boolean BigQuery SQL expression expressing whether a single row in telemetry.main would have
+   * matched the audience definition. These are used for sizing experiment populations,
+   * not for targeting.
+   */
+  desktop_telemetry?: string;
 }
 
 const audiences :  {[id: string] : Audience } = {
   all_english: {
     name: "All English users",
     description: "All users in en-* locales.",
-    targeting: "localeLanguageCode == \"en\""
+    targeting: "localeLanguageCode == \"en\"",
```

Comment:
Does the targeting string work with nested single quotes? (similar to how we've nested single quotes in the `desktop_telemetry` string below to avoid escaping.)

ie: `"localeLanguageCode == 'en'"` or even ``` `localeLanguageCode == "en"` ```

Reply:
the apache jexl spec says single-quoted string literals are allowed and normandy devtools seems to accept them so i'll assert "yes"

Reply:
Quick note that although we do allow single quotes, Apache JEXL is not the JEXL we use. That is a different language entirely (that turns out to be not suitable for the original goals of filter expressions). Specifically Apache's JEXL is "Java Expression Language". We use "Javascript Expression Language", which we have forked here: https://github.com/mozilla/mozjexl.

It's an unfortunate name collision, the two languages share only a name, not a history.